docs/service-catalog/docs/020-architecture-service-catalog.md

```diff
@@ -2,16 +2,14 @@
 title: Architecture
 ---
 
-The diagram and steps describe the basic Service Catalog workflow:
+The diagram and steps describe the Service Catalog workflow and the roles of specific cluster and Environment-wide resources in this process:
 
-![Service Catalog flow](assets/service-catalog-flow.png)
+![Service Catalog flow](assets/service-catalog-flow.svg)
 
-1. During the Kyma installation, the system registers the default Service Brokers in the Kyma cluster. The cluster administrator can manually register other Service Brokers in the Kyma cluster.
+1. The Kyma installation results in the registration of the default Service Brokers in the Kyma cluster. The Kyma administrator can manually register other ClusterServiceBrokers in the Kyma cluster. The Kyma user can also register a Service Broker (ServiceBroker) in a given Environment.
```

Comment:
"The Kyma user can also register a Service Broker (ServiceBroker) in a given Environment." -> Is there any need to double Service Broker here?